### ISSUE TITLE
FileSystemUriFetcher doesn't allow fetching files from the current directory

### ISSUE THREAD

**Issue by kpreid** [google/caja#1215]
Original [issue 1215](https://code.google.com/p/google-caja/issues/detail?id=1215) created by jasvir on 2010-05-27T22:31:15.000Z:

<b>What steps will reproduce the problem?</b>

I am attempting to cajole the mootools framework by including it in an
html document. When trying to cajole a simple xhtml document with only
a single div in the body and script tag referencing an external JS
file in the head, I get a warning that says caja &quot;failed to load
external uri&quot;.

Console output can be found here: http://pastebin.com/raw.php?i=RYJwmMfc
The document, mootools.htm, can be found here: http://pastebin.com/raw.php?i=cYwhZ94W
The JS file is in the same directory as the mootools.htm file.

tl;dr How do I make caja successfully load a JS file referenced in an
(X)HTML file?
